Arrowgene.Ddon.Shared/Entity/PacketStructure/C2SConnectionPingReq.cs

```diff
@@ -0,0 +1,28 @@
+using Arrowgene.Buffers;
+using Arrowgene.Ddon.Shared.Network;
+
+namespace Arrowgene.Ddon.Shared.Entity.PacketStructure
+{
+    public class C2SConnectionPingReq : IPacketStructure
```

Comment:
Should document that this is expected to be requested by the client every 5sec here or in the handler.